node-mapnik/scripts/test

```diff
@@ -27,13 +27,13 @@ PGUSER=postgres psql -d template_postgis -c "GRANT ALL ON spatial_ref_sys TO PUB
 
 # Link previously built node-mapnik
 cd node-mapnik
-npm link
+sudo npm link
```

Comment:
this is intended to run in a vagrant machine. I'm aware of other approaches but the point here is to run this w/o manual intervention and it does NOT work as it is.